### PR TITLE
Add Nginx

### PR DESCRIPTION
**Pull request by radamson**
Adds nginx to support deploying behind a load balancer that does SSL termination. Nginx is configured to provide the `X-Forwarded` type headers to rails which address `ActionController::InvalidAuthenticityToken (ActionController::InvalidAuthenticityToken):` errors.

**References:**
* https://github.com/rails/rails/issues/22965

### CODE REVIEW

**Comment by radamson** [comment]
I forgot to update the README with instructions on how to run. I'll do that shortly.

**Comment by mickohanlon23** [comment]
Mention reference this PR in Issue #89 after merging.

**Comment by radamson** [comment]
Check that `iis` is populated when running locally, with docker, and with docker compose.

**Comment by mickohanlon23** [comment]
> Check that `iis` is populated when running locally, with docker, and with docker compose.

Confirmed that `iss` is populated to `http://localhost:3000` when running all 3 ways.

File: app/views/health_cards/show.pdf.erb [hunk 1]
```diff
@@ -87,7 +87,9 @@
         <% @qr_codes.chunks.each do |ch| %>
           <div class="tile is-parent">
             <div class="tile is-child box has-text-centered">
-              <%= wicked_pdf_image_tag patient_qr_code_url(@patient, ch.ordinal, format: :png) %>
+              <%= wicked_pdf_image_tag url_for(controller: 'qr_codes', action: 'show', host: "localhost",
+                                               protocol: 'http', port: ENV['PORT'], patient_id: @patient.id,
```
Comment (mickohanlon23) [inline app/views/health_cards/show.pdf.erb:91]: It looks like this causes the QR Code in the PDF to not render when running using Docker (it still works fine when running with docker-compose).

I was able to fix this by adding `ENV PORT=3000` to `Dockerfile`.

**Review by mickohanlon23 — CHANGES_REQUESTED** [review]

File: app/views/health_cards/show.pdf.erb (hunk 1 above)
Comment (radamson) [reply app/views/health_cards/show.pdf.erb:91]: Good catch. Fixed in https://github.com/dvci/health_cards/pull/97/commits/42ad67f0a9ee6791cc17a9194ec458ecbd3f6349.

Comment (mickohanlon23) [reply app/views/health_cards/show.pdf.erb:91]: QR Code properly renders now!

**Review by mickohanlon23 — APPROVED** [review]
All functionality works when running locally, using Docker, and using Docker-compose. 

Most recent commit seems to successfully address Issue #94. I tested passing in different HOST environment variables to the docker container, as well as not passing in any, and neither scenario produced errors. 👍

**Pull request merged**